### PR TITLE
#143412429 Update Http to Https

### PR DESCRIPTION
**Pull request by CeciliaWanjiku**
#### What does this PR do? 
It updates the the http paths to https so that errors are not raised when deploying to heroku
 #### Description of Task to be completed?
Deploying application to heroku
 #### How should this be manually tested?
Opening the application using the heroku link
 #### Any background context you want to provide?
 #### What are the relevant pivotal tracker stories?
143412429
 #### Screenshots (if appropriate)
 #### Questions:

### CODE REVIEW

**Comment by gngchrs** [comment]
minor: PR doesn't follow correct format. Title n all

I'm not seeing the update to the `app.json` file.

**Comment by gangachris** [comment]
minor: PR doesn't follow correct format. Title n all

I'm not seeing the update to the `app.json` file.

**Comment by CeciliaWanjiku** [comment]
@ganga Sorry I took long, but i have implemented the feedback

File: app/views/includes/foot.jade [hunk 1]
```diff
@@ -39,4 +39,4 @@ script(type='text/javascript', src='/js/init.js')
 script(type='text/javascript', src='/socket.io/socket.io.js')
 
 //Livereload script rendered
-//script(type='text/javascript', src='http://localhost:35729/livereload.js')
\ No newline at end of file
+//script(type='text/javascript', src='https://localhost:35729/livereload.js')
```
Comment (LWanjiru) [inline app/views/includes/foot.jade:42]: Add a new line here.  (to resolve the red sign at the end of the line). All else appears to be well

**Review by LWanjiru — APPROVED** [review]
Minor change needed, good to merge

**Pull request merged**